<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="GauriShankar" failures="1" tests="4" name="Test" time="11.631" errors="0" timestamp="2024-08-12T22:32:23 IST">
  <testcase classname="api.Testcases.UserTestcases2" name="testCreateUser" time="4.253"/>
  <testcase classname="api.Testcases.UserTestcases2" name="testGetUser" time="1.283"/>
  <testcase classname="api.Testcases.UserTestcases2" name="testUpdateUser" time="2.636"/>
  <testcase classname="api.Testcases.UserTestcases2" name="testdeleteUser" time="1.556">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.Testcases.UserTestcases2.testdeleteUser(UserTestcases2.java:127)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- testdeleteUser -->
</testsuite> <!-- Test -->
